### PR TITLE
OCPBUGS-27842: Add sno section to alert description

### PR DESCRIPTION
**Pull request by qJkee**
Add additional information to HighOverallControlPlaneCPU alert description regarding SNO clusters.

### CODE REVIEW

File: bindata/assets/alerts/cpu-utilization.yaml [hunk 1]
```diff
@@ -14,11 +14,15 @@ spec:
...
             description: >-
-              Given three control plane nodes, the overall CPU utilization may only be about 2/3 of all available capacity.
+              On a multi-node cluster with three control plane nodes, the overall CPU utilization may only be about 2/3 of all available capacity.
               This is because if a single control plane node fails, the remaining two must handle the load of the cluster in order to be HA.
-              If the cluster is using more than 2/3 of all capacity, if one control plane node fails, the remaining two are likely to
-              fail when they take the load.
+              If the cluster is using more than 2/3 of all capacity, if one control plane node fails, the remaining two are likely to fail when they take the load.
               To fix this, increase the CPU and memory on your control plane nodes.
+              
+              On a SNO cluster, this alert will also fire if the 2/3 of the CPU cores of on the SNO are in use by any workload. This level of CPU utlization
```
Comment (simonpasquier) [inline bindata/assets/alerts/cpu-utilization.yaml:22]: (suggestion) define SNO acronym first
```suggestion
              On a single node OpenShift (SNO) cluster, this alert will also fire if the 2/3 of the CPU cores of the node are in use by any workload. This level of CPU utlization
```

File: bindata/assets/alerts/cpu-utilization.yaml [hunk 2]
```diff
@@ -14,11 +14,15 @@ spec:
...
+              of an SNO cluster is probably not a problem under most circumstances, but high levels of utilization may result in degraded performance.
+              To manage this alert and turn off it in case of false positives see the following link: 
+              https://docs.openshift.com/container-platform/4.14/monitoring/managing-alerts.html
```
Comment (simonpasquier) [inline bindata/assets/alerts/cpu-utilization.yaml:25]: Are we going to update the 4.14 path? One option would be to use `https://docs.openshift.com/container-platform/latest/monitoring/managing-alerts.html` but the page might move in the future.

File: bindata/assets/alerts/cpu-utilization.yaml [hunk 3]
```diff
@@ -14,11 +14,15 @@ spec:
...
+              On a multi-node cluster with three control plane nodes, the overall CPU utilization may only be about 2/3 of all available capacity.
               This is because if a single control plane node fails, the remaining two must handle the load of the cluster in order to be HA.
-              If the cluster is using more than 2/3 of all capacity, if one control plane node fails, the remaining two are likely to
-              fail when they take the load.
+              If the cluster is using more than 2/3 of all capacity, if one control plane node fails, the remaining two are likely to fail when they take the load.
               To fix this, increase the CPU and memory on your control plane nodes.
+              
+              On a SNO cluster, this alert will also fire if the 2/3 of the CPU cores of on the SNO are in use by any workload. This level of CPU utlization
+              of an SNO cluster is probably not a problem under most circumstances, but high levels of utilization may result in degraded performance.
+              To manage this alert and turn off it in case of false positives see the following link:
```
Comment (simonpasquier) [inline bindata/assets/alerts/cpu-utilization.yaml:24]: "turn off" is a bit vague: are we advising to silence the alert or dropping it with the AlertRelabeling CRD?

**Review by simonpasquier — COMMENTED** [review]
The summary annotation only needs an update.

File: bindata/assets/alerts/cpu-utilization.yaml (hunk 3 above)
Comment (vrutkovs) [reply bindata/assets/alerts/cpu-utilization.yaml]: I think we want to suggest to silence it

**Review by simonpasquier — COMMENTED** [review]
I think that the summary annotation needs an update too. It mentions "three control plane nodes".

File: bindata/assets/alerts/cpu-utilization.yaml [hunk 4]
```diff
@@ -11,14 +11,18 @@ spec:
           annotations:
             summary: >-
               CPU utilization across all three control plane nodes is higher than two control plane nodes can sustain; a single control plane node outage may
-              cause a cascading failure; increase available CPU.
+              cause a cascading failure; For Single Node OpenShift (SNO) clusters, monitor CPU usage, as high levels may impact performance. increase available CPU.
```
Comment (simonpasquier) [inline bindata/assets/alerts/cpu-utilization.yaml:14]: From my POV the summary is a bit lengthy. From https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md:

>  Think of summary as the first line of a commit message, or an email subject line. It should be brief but informative. The description is the longer, more detailed explanation of the alert.

````
CPU utilization across all control plane nodes is more than 60% of the total available CPU.

Comment (simonpasquier) [reply bindata/assets/alerts/cpu-utilization.yaml:14]: sgtm

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Comment by sdodson** [comment]
Could we not have an alert that's tuned for SNO instead?

**Pull request merged**